src/api/remediations.js

```diff
@@ -0,0 +1,19 @@
+const API_BASE = '/r/insights/platform/remediations/v1/remediations';
```

Comment:
Consider passing API_BASE from outside so consumers can pass their own API. But that's for the future.